### PR TITLE
Created readme file

### PR DESCRIPTION
**Pull request by mpsujek**

### CODE REVIEW

File: js/readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# This is a project to keep all my codewars solutions.
```
Comment (Gelio) [inline js/readme.md]: While this readme is fine, it does not look like other readmes :slightly_smiling_face: 

Readmes generally consist of:
1. A title
2. Some description

In your case, there is no concrete title. The description is made up of headers. Let's make it regular text :slightly_smiling_face:

Take a look at readmes of other GitHub projects and try to adjust this one

**Comment by Gelio** [comment]
Btw. nice idea to add a readme :smile:

File: js/readme.md (hunk 1 above)
Comment (mpsujek) [reply js/readme.md]: I've made changes

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#Codewars Solutions Project
```
Comment (Gelio) [inline readme.md]: Missing space 

```suggestion
# Codewars Solutions Project
```

Comment (mpsujek) [reply readme.md]: I will fix that

**Comment by mpsujek** [comment]
> Nice 👍 Btw. do you think that _changes after review_ is a proper commit name :>?

No it shoud be more complex

**Pull request merged**